### PR TITLE
Add initial Sentry integration

### PR DESCRIPTION
**Pull request by Craftplacer**

### CODE REVIEW

File: SimpleClassicThemeTaskbar/UI/SystemTray/SystemTrayIcon.cs [hunk 1]
```diff
@@ -35,11 +35,7 @@ public Icon Icon
             get => Icon.FromHandle(((Bitmap)Image).GetHicon());
             set
             {
-                try
-                {
-                    Image = value.ToBitmap();
-                }
-                catch { }
```
Comment (Craftplacer) [inline SimpleClassicThemeTaskbar/UI/SystemTray/SystemTrayIcon.cs:42]: I should undo this change

File: SimpleClassicThemeTaskbar/Forms/CrashForm.cs [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+        private void DescriptionLinkLabel_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
+        {
+            MessageBox.Show(this,
+                @"The currently known data collected by error reports sent in debug builds is:
+
+• Current application log
+    • OS version
+    • SCT and application version
+    • Potentially currently running depending on debug output
```
Comment (AEAEAEAE4343) [inline SimpleClassicThemeTaskbar/Forms/CrashForm.cs:104]: This is vague

File: SimpleClassicThemeTaskbar/Helpers/Logger.cs [hunk 3]
```diff
@@ -41,12 +46,17 @@ public static void Initialize(LoggerVerbosity verbosity)
...
-            fs = new FileStream("latest.log", FileMode.Create, FileAccess.Write, FileShare.Read);
+            fs = new FileStream(FilePath = "latest.log", FileMode.Create, FileAccess.Write, FileShare.Read);
+            sw = new StreamWriter(fs) { AutoFlush = true };
             Log(LoggerVerbosity.Basic, "Logger", "Succesfully initialized logger");
 
             Log(LoggerVerbosity.Detailed, "SystemDump", "Performing quick system dump");
             Log(LoggerVerbosity.Detailed, "SystemDump", $"OS: {RuntimeInformation.OSDescription} {RuntimeInformation.OSArchitecture}");
-            if (ApplicationEntryPoint.SCTCompatMode) Log(LoggerVerbosity.Detailed, "SystemDump", $"SCT version: {Assembly.LoadFrom("C:\\SCT\\SCT.exe").GetName().Version}");
+
+            // HACK: This uses a fixed path, this needs to be replaced
```
Comment (AEAEAEAE4343) [inline SimpleClassicThemeTaskbar/Helpers/Logger.cs:56]: Don't think we need to replace this. When SCTCompatMode == true it's safe to assume that SCT will always be located in C:\SCT\SCT.exe.

File: SimpleClassicThemeTaskbar/UI/SystemTray/SystemTrayIcon.cs (hunk 1 above)
Comment (AEAEAEAE4343) [reply SimpleClassicThemeTaskbar/UI/SystemTray/SystemTrayIcon.cs:42]: You should.

**Review by AEAEAEAE4343 — APPROVED** [review]
Seems fine. Though as you said, you should revert that change in SystemTrayIcon.cs but we can do that after the merge.

File: SimpleClassicThemeTaskbar/Forms/CrashForm.cs [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+• Stack trace (information about where the program crashed)
+
+• System Environment
+    • OS version (e.g. Microsoft Windows 10.0.22000)
+    • .NET runtime version
+    • SCT application version
+    • Senty SDK version (Error reporting SDK version)
```
Comment (AEAEAEAE4343) [inline SimpleClassicThemeTaskbar/Forms/CrashForm.cs:112]: Senty?

File: SimpleClassicThemeTaskbar/Helpers/Logger.cs (hunk 3 above)
Comment (Craftplacer) [reply SimpleClassicThemeTaskbar/Helpers/Logger.cs:56]: Fair, this should be addressed in SCT generally though. A fixed installation path is not good.

File: SimpleClassicThemeTaskbar/Forms/CrashForm.cs (hunk 2 above)
Comment (Craftplacer) [reply SimpleClassicThemeTaskbar/Forms/CrashForm.cs:104]: What can I do to improve it?

File: SimpleClassicThemeTaskbar/Forms/CrashForm.cs (hunk 4 above)
Comment (Craftplacer) [reply SimpleClassicThemeTaskbar/Forms/CrashForm.cs:112]: *Sentry, oops

File: SimpleClassicThemeTaskbar/Helpers/Logger.cs (hunk 3 above)
Comment (AEAEAEAE4343) [reply SimpleClassicThemeTaskbar/Helpers/Logger.cs:56]: Maybe when we have libsct ready this will be easier to manage? For now I don't see an easy way to resolve it.

File: SimpleClassicThemeTaskbar/Forms/CrashForm.cs (hunk 2 above)
Comment (AEAEAEAE4343) [reply SimpleClassicThemeTaskbar/Forms/CrashForm.cs:104]: 'Depending on the problem, this might also include information about open windows'

Comment (Craftplacer) [reply SimpleClassicThemeTaskbar/Forms/CrashForm.cs:104]: I notice my head skipped a word again when writing this, I see.

**Review by AEAEAEAE4343 — CHANGES_REQUESTED** [review]
Now that I think of it, you should probably fix these before merging:

- [x] Senty -> Sentry
- [ ] Undo the SystemTrayIcon change
- [x] Fix your wording in CrashForm

Just amend the commit

File: SimpleClassicThemeTaskbar/Helpers/Logger.cs (hunk 3 above)
Comment (Craftplacer) [reply SimpleClassicThemeTaskbar/Helpers/Logger.cs:56]: I'll remove the comment for now, but we'll have to keep track of this.

**Comment by Craftplacer** [comment]
I just committed changes, re-review them and then I think this PR is good to go

**Review by AEAEAEAE4343 — APPROVED** [review]

**Comment by AEAEAEAE4343** [comment]
Looks good to meee
